ui/src/components/confirmation_modal/ConfirmationModal.scss

```diff
@@ -0,0 +1,3 @@
+.euiModalBody__overflow {
+    -webkit-mask-image: none !important;
```

Comment:
Sometimes the content of the popup will become red, i dont know why, previously the content were covered by the mask image fill, for now i cant rpelicate that problem again, but if i dont add the css, it will become like this
![16844695526311684469552634](https://github.com/caraml-dev/turing/assets/58201002/c4b2db57-c9ef-456e-b4f3-fb3437543971)